tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -754,7 +758,7 @@ task formatPipelineOutputs {
   >>>
 
   runtime {
-    docker: "broadinstitute/horsefish:twisttcap_scripts"
+    docker: "broadinstitute/horsefish:tdr_import_v1.1"
```

Comment:
this is the same code as before, we just decided to make versioned tags to be safer with updates.